filters/quantitative_ques/filter.py

```diff
@@ -0,0 +1,24 @@
+import spacy
+import sys
+sys.path.append('/home/tanay/Documents/GEM/NL-Augmenter')
```

Comment:
I don't think we can have hardcoded paths like this, especially those bearing your name, going into the final code. I am not sure why you need sys.path.append, but if you do, try to make it work in a way that repo-relative paths work..

Reply:
Sorry for that, have fixed it in the latest commit